packages/calypso-color-schemes/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Release Notes
+
+## 2.0.0
+
+This update brings backwards-incompatible changes to the package. The most notable ones include:
+
+* Updated [Color Studio](https://color-studio.blog), the primary dependency, to the most recent version.
+* Following the [files](https://github.com/Automattic/color-studio/blob/master/dist/color-properties.css) [generated](https://github.com/Automattic/color-studio/blob/master/dist/color-properties-rgb.css) by Color Studio, updated all properties to use consistent names (including index numbers).
+* Phased out all SCSS variables in favor of CSS custom properties.
```

Comment:
Why did this happen? This breaks dependent SASS pipelines. Importing CSS is very different from importing SASS in that it generates output.

I'd speculate that this is the cause of https://github.com/Automattic/wp-calypso/pull/35949#issuecomment-531599278 and is likely related to a bump in bundle sizes @sgomes has been tracking down.

Reply:
As I already mentioned in one of the comments, it was a handful of the _remaining_ SCSS variables that were phased out. Those had relied on SCSS color functions like `darken` or `lighten`, and that was the reason why the original PR introducing `calypso-color-schemes` left them out.

In other words, I completed the switch from one SCSS variables to CSS custom properties that had already been done in 95%.